packages/radix-ui-themes/src/components/theme-panel.tsx

```diff
@@ -23,605 +24,628 @@ import { themePropDefs } from '../props/index.js';
 import type { ComponentPropsWithout, RemovedProps } from '../helpers/index.js';
 import type { GetPropDefTypes } from '../props/index.js';
 
-interface ThemePanelProps extends Omit<ThemePanelImplProps, keyof ThemePanelImplPrivateProps> {
+const keyboardInputElement = `
+  [contenteditable],
+  [role="combobox"],
+  [role="listbox"],
+  [role="menu"],
+  input:not([type="radio"], [type="checkbox"]),
+  select,
+  textarea
+`;
+
+/** Listen to keydown events and fire a callback when a hotkey is pressed */
+const useHotKey = (
+  key: string | null | false,
+  callback: () => void,
+  options?: {
+    /** Determines whether the hotkey listener is active or not */
+    enabled?: boolean;
+  }
+) => {
```

Comment:
Let's put the hook and the input elements selector back into the component body; it's easier to read code when things are declared close to where they are used

Reply:
It doesn't make sense to inline this effect into the component's body because you use it in two different places. You'll end up with two chunks of the same code.

Plus, it's located in the same file, just above where it's used, and the name is self-explanatory in this case. That should be good enough.

I can move the `keyboardInputElement` declaration into the useHotKey hook's body if you want.

Reply:
> It doesn't make sense to inline this effect into the component's body because you use it in two different places. You'll end up with two chunks of the same code.

Oh I see, you are right, I missed that